frame/support/src/storage/migration.rs

```diff
@@ -195,3 +197,154 @@ pub fn take_storage_item<K: Encode + Sized, T: Decode + Sized, H: StorageHasher>
 ) -> Option<T> {
 	take_storage_value(module, item, key.using_encoded(H::hash).as_ref())
 }
+
+/// Move a storage from a pallet prefix to another pallet prefix.
+///
+/// NOTE: It actually moves every key after the storage prefix to the new storage prefix,
+/// key on storage prefix is also moved.
+pub fn move_storage_from_pallet(item: &[u8], old_pallet: &[u8], new_pallet: &[u8]) {
```

Comment:
Maybe you could add some example and more docs on what these parameters are?

This is relative complicated to understand from the current docs.

Reply:
indeed, I improved, I hope its clearer now.